interpreter/interpreter_test.go

```diff
@@ -218,30 +210,6 @@ func TestEval(t *testing.T) {
             plusX(r:2) == 7 or fail()
 			`,
 		},
-		{
-			name: "scope closing mutable",
-			query: `
-			x = 5
-            plusX = (r) => r + x
-            plusX(r:2) == 7 or fail()
-			x = 1
-            plusX(r:2) == 3 or fail()
-			`,
-		},
-		{
-			name: "nested scope mutations not visible outside",
-			query: `
-			x = 5
-            xinc = () => {
-                x = x + 1
-                return x
-            }
-            xinc() == 6 or fail()
-            x == 5 or fail()
-            x = 1
-            xinc() == 2 or fail()
-			`,
-		},
```

Comment:
Instead of deleting these tests can you update them to test the opposite behavior?